extensions/amp-story/1.0/amp-story.js

```diff
@@ -345,6 +345,10 @@ export class AmpStory extends AMP.BaseElement {
     // prerendering, because of a height incorrectly set to 0.
     this.mutateElement(() => {});
 
+    if (!this.win.CSS.supports || !this.win.CSS.supports('height', '100dvh')) {
```

Comment:
We can use `?.` in source code (but not in transformer's script):

```suggestion
    if (!this.win.CSS?.supports?.('height: 1dvh')) {
```